test/github_test.dart

```diff
@@ -435,30 +435,32 @@ void main() {
     });
   });
 
-  test("pkg-github-macos uploads standalone Mac OS archives", () async {
+  test("pkg-github-macos-x64 uploads standalone Mac OS x64 archives", () async {
```

Comment:
We should also test arm64 for Mac OS and Linux.

Reply:
Added two arm64 test cases.